lib/components/home/artwork_rails/artwork_rail.js

```diff
@@ -155,10 +155,15 @@ class ArtworkRail extends React.Component {
 
   render() {
     const sideMargin = isPad ? 40 : 20
+    const style = { marginLeft: sideMargin, marginRight: sideMargin }
+    if (!this.props.relay.variables.fetchContent) {
+      // if there's no content, set minHeight to prevent spinners from all the rails showing
+      style.minHeight = 500
```

Comment:
I didn’t realise you were going with a real static value here. How about using the height of the window?